### DISCUSSION TITLE
Double Payment Intents

### DISCUSSION THREAD

**Discussion by thevanitylab** [swellstores/community#568] [Help]
Really urgently need assistance on this. Store just went live, and despite normal behaviour in testing mode we are seeing double payment intents in the live store. I can see that this came up earlier this year and according to [this discussion](https://github.com/orgs/swellstores/discussions/478) a fix went out. 

Stripe is the payment processor, and currently also have Google Pay and Apple Pay activated.

Custom storefront using 
nextjs 14.2.5
swell-js 4.2.0
swell-node: 5.6.0

Using `swell.payment.createElements({...})`, then `swell.payment.tokenize({})`, then `swell.cart.submitOrder()`.

**Comment by SergeySwell** [comment]
@thevanitylab could you please contact Swell support and provide a link to the order with double payment intent?

**Reply by thevanitylab** [reply]
Will do

**Comment by logeshswell** [comment]
I hope your query has been resolved through Swell Support.
So, I am closing this discussion and please feel free to reopen it back if you have follow-up queries.
